glue/plugins/__init__.py

```diff
@@ -3,10 +3,17 @@
 from . import ginga_viewer
 
 
-def load_all_plugins():
+def register_plugins():
+    from ..config import qt_client, exporters
+    qt_client.add_plugin('glue.plugins.ginga_viewer')
+    exporters.add_plugin('glue.plugins.export_d3po')
+    exporters.add_plugin('glue.plugins.export_plotly')
+    
+
+def load_plugin(plugin):
     """
-    Load built-in plugins
+    Load plugin referred to by name 'plugin'
     """
-
-    from .ginga_viewer import load_ginga_viewer_plugin
-    load_ginga_viewer_plugin()
+    import importlib
+    module = importlib.import_module(plugin)
+    module.load_plugin()
```

Comment:
Probably best to give an informative error message here if the plugin-author didn't create a `load_plugin` function.